templates/components/draft_results.html

```diff
@@ -0,0 +1,27 @@
+{% macro draftResults(selections) -%}
+    <!-- The full draft board. can we make this a max height and then have a scrollbar? -->
+    Picks
+    <table class='table' style="height: 400px; display: flex; width: 100%; flex-flow: column;">
```

Comment:
should we move this over to a css class? ditto for other inline stylings

(i realize some of this may have been preexisting)